cluster/examples/kubernetes/noobaa/noobaa-backing-store-example-aws-s3.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: noobaa.rook.io/v1alpha1
```

Comment:
nit: since these yaml files have parent dirs for "examples" and "noobaa", you could simplify the filenames to something like backing-store-aws-s3.yaml, similar to other storage provider yamls.

Reply:
I am not sure - I think that many rook users will take a release from operatorhub.io and will not clone the project, so they will download these yaml examples individualy or with wget/curl. This is why I wanted to have the noobaa prefix so that the downloaded file names will have a standalone name. Makes sense?

Reply:
Other storage providers have simple names such as `operator.yaml` that aren't standalone. All the docs around a storage provider discuss only those storage providers, so it just seems redundant. It makes tab completion more difficult for the common scenario if there are many files with the same prefix. :) But that's fine to keep it this way if preferred.

Reply:
I think I prefer to start with the prefix. 
Will rename if it becomes a nuisance.
Resolving.